<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>731</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>193</green>
        <blue>193</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>193</green>
        <blue>193</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>193</green>
        <blue>193</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>193</green>
        <blue>193</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>MS Reference Sans Serif</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="commandLinkButton_admin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>236</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accès à l'administration</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_stock">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>406</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accès aux stocks</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_caisse">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accès à la caisse</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>227</red>
          <green>28</green>
          <blue>13</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>227</red>
          <green>28</green>
          <blue>13</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>227</red>
          <green>28</green>
          <blue>13</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
